src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  font-family: inherit;
+  padding: 0;
+}
+
+.search-bar {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin: 20px 0;
+  position: relative;
+  cursor: pointer;
+}
+
+.search-bar--small {
+  height: 42px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin: 20px 0;
+  position: relative;
+  cursor: pointer;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus-within,
+.search-bar--small:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f7f7f8 100%);
+}
+
+.search-bar__icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.search-bar--small .search-bar__icon {
```

Comment:
Avoid nesting selector, it's better to create specific class for this icon
like `.search-bar__icon--small`